miden-lib/src/errors/mod.rs

```diff
@@ -0,0 +1,3 @@
+#[rustfmt::skip]
+mod tx_kernel_errors;
+pub use tx_kernel_errors::*;
```

Comment:
Would it make sense to export the module itself? Basically like:
```rust
#[rustfmt::skip]
pub mod tx_kernel_errors;
```
This would make error constants more explicit, which I think is a good thing.

Reply:
I did this originally but the fully qualified path for an error code seemed a bit long and a bit redundant as well, but I can change it back

Reply:
Changed this